packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -416,4 +411,15 @@ export const run = async () => {
   if (requiredEnvVars.RUN_STATE_BATCH_SUBMITTER) {
     loop(() => stateBatchSubmitter.submitNextBatch())
   }
+
+  if (
+    config.bool('run-prometheus-server', env.RUN_PROMETHEUS_SERVER === 'true')
+  ) {
+    // Initialize metrics server
+    await createMetricsServer({
+      logger,
+      registry: metrics.registry,
+      port: config.uint('prometheus-port'),
```

Comment:
hostname needs to be passed through here as well

Reply:
What do you think about making these options `metrics-server-*`? Just because I don't think they are technically coupled to prometheus, are they?

`hostname: config.str('metrics-server-hostname', '127.0.0.1')`

Reply:
^as of now they are! lol i switched because of this comment: https://github.com/ethereum-optimism/optimism/pull/792#discussion_r631310643 not opinionated about this but i think i'll stick with prometheus since our client is basically a wrapper